Packs/Humio/Integrations/Humio/Humio.py

```diff
@@ -0,0 +1,392 @@
+    else:
+        raise ValueError(
+            "Error:"
+            + " response from server was: "
+            + str(response.text)
+        )
+
+
+def humio_delete_alert(client, args, headers):
+    data: Dict[str, str] = {}
+    url = "/api/v1/repositories/" + args.get("repository") + "/alerts/" + args.get("id")
+    headers["Accept"] = "application/json"
+    response = client.http_request("DELETE", url, data, headers)
+    if response.status_code == 204:
+        return ("Command executed. Status code " + str(response), None, None)
+    else:
+        raise ValueError(
+            "Error:"
+            + " response from server was: "
+            + str(response.text)
+        )
+
+
+def humio_list_notifiers(client, args, headers):
+    data: Dict[str, str] = {}
+    url = "/api/v1/repositories/" + args.get("repository") + "/alertnotifiers"
+    headers["Accept"] = "application/json"
+    response = client.http_request("GET", url, data, headers)
+    if response.status_code == 200:
+        result = response.json()
+        markdown = tableToMarkdown("Humio Notifiers", result, removeNull=True)
+        outputs = {"Humio.Notifier(val.id == obj.id)": result}
+        return markdown, outputs, result
+    else:
+        raise ValueError(
+            "Error:"
+            + " response from server was: "
+            + str(response.text)
+        )
+
+
+def humio_get_notifier_by_id(client, args, headers):
+    data: Dict[str, str] = {}
+    url = (
+        "/api/v1/repositories/"
+        + args.get("repository")
+        + "/alertnotifiers/"
+        + args.get("id")
+    )
+    headers["Accept"] = "application/json"
+    response = client.http_request("GET", url, data, headers)
+    if response.status_code == 200:
+        if not response.text:
+            raise ValueError("Notifier with id " + str(args.get("id")) + " not found")
+        result = response.json()
+        markdown = tableToMarkdown("Humio Notifiers", result, removeNull=True)
+        outputs = {"Humio.Notifier(val.id == obj.id)": result}
+        return markdown, outputs, result
+    else:
+        raise ValueError(
+            "Error:"
+            + " response from server was: "
+            + str(response.text)
+        )
+
+
+def fetch_incidents(client, headers):
+    incidentquery = demisto.params().get("queryParameter")
+    incidentrepo = demisto.params().get("queryRepository")
+    timestampfrom = demisto.params().get("queryStartTime")
+    lastrun = demisto.getLastRun()
+    url = "/api/v1/repositories/" + incidentrepo + "/query"
+    headers["Accept"] = "application/json"
+    try:
+        if lastrun["time"]:
+            data = {}
+            data["queryString"] = incidentquery
+            data["start"] = int(lastrun["time"]) * 1000
+            data["end"] = "now"
+            data["isLive"] = False
+            data["timeZoneOffsetMinutes"] = int(
+                demisto.params().get("queryTimeZoneOffsetMinutes")
```

Comment:
You can look at HelloWorld example, we recommend moving API functions into the Client, and call client functions in `_command` functions.
This way you can reuse code. In this case I see that you could do `client.query(..)`